Infrastructure/Services/ProcessEmailsService.cs

```diff
@@ -0,0 +1,450 @@
+﻿using Application.Commands.Enquiry.Build;
+using Application.Common.DTO;
+using Application.Common.Interfaces;
+using Application.Common.Models.Admin;
+using Application.Extensions;
+using Domain;
+using Domain.Enums;
+using Infrastructure.Configuration;
+using Infrastructure.Mapping.Configuration;
+using Mapster;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using EmailStatus = Domain.Enums.EmailStatus;
+
+namespace Infrastructure.Services;
+
+public class ProcessEmailsService : IProcessEmailsService
+{
+    private const string ScheduleName = "Process Emails";
+    private const string TestExtraInfoKey = "test_extra_info";
+
+    private readonly IUnitOfWork _unitOfWork;
+    private readonly ILogger<AddEnquiryCommandHandler> _logger;
```

Comment:
It should be `private readonly ILogger<ProcessEmailsService> _logger;`

Reply:
Good spot, done